sklearn/svm/classes.py

```diff
@@ -48,16 +48,16 @@ class LinearSVC(BaseEstimator, LinearClassifierMixin,
         two classes.
         `ovr` trains n_classes one-vs-rest classifiers, while `crammer_singer`
         optimizes a joint objective over all classes.
-        While `crammer_singer` is interesting from an theoretical perspective
-        as it is consistent it is seldom used in practice and rarely leads to
-        better accuracy and is more expensive to compute.
+        While `crammer_singer` is as interesting from an theoretical perspective
+        as it is consistent, it is seldom used in practice and rarely leads to
+        better accuracy. It is also more expensive to compute.
```

Comment:
I think it should be "as it rarely leads to better accuracy and is more expensive to compute".